### DISCUSSION TITLE
Constraints between chains for long skirts?

### DISCUSSION THREAD

**Discussion by KevlarIRA** [pafuhana1213/KawaiiPhysics#92] [Ideas - 要望・アイデア]
hello pafuhana1213,

I was wondering would it be possible to have some sort of constraint between bone chains/ or any sustainable system to help avoid clipping on long skirts since currently right now it is very hard to make a system which stops the legs from clipping through the skirt if the length is longer than the knees

**Comment by KevlarIRA** [comment]
@pafuhana1213 is it possible for a feature like this to exist the another project on github called SPCR joint dynamics has something similar to this but isnt updated for 5.2.1 just 5.2

**Comment by pafuhana1213** [comment]
Hi guys,
this update may...may help you. maybe
https://github.com/pafuhana1213/KawaiiPhysics/discussions/112

thanks.
